src/components/Navbar.js

```diff
@@ -12,43 +9,63 @@ const argumentsSet = [
   ["Investors", "For investors"]
 ];
 
+//WARNING: the wheel was invented here!!!
 class Navbar extends React.Component {
   constructor(props) {
     super(props);
     this.state = {activeLinkName: argumentsSet[0][0]};
   }
+  componentDidMount() {
+    window.addEventListener("scroll", () => this.onWindowScroll(), true);
```

Comment:
[1/2 - listener] plz use `window.addEventListener("scroll", this.onWindowScroll, true);`

because `removeEventListener` expects same listener (`a === b`), and `() => this.onWindowScroll() !==  this.onWindowScroll // true`